packages/internal/src/__tests__/nestedPages.test.ts

```diff
@@ -1,15 +1,26 @@
 import path from 'path'
 
+import { expect } from '@jest/globals'
+
 import { getPaths } from '@redwoodjs/project-config'
 
 import { prebuildWebFile } from '../build/babel/web'
 import { cleanWebBuild } from '../build/web'
 
 const FIXTURE_PATH = path.join(__dirname, 'fixtures/nestedPages')
 
+function normalizeStr(str: string) {
+  return str
+    .split('\n')
+    .map((line) => line.trim())
+    .join('\n')
+    .trim()
+}
```

Comment:
I was getting failures for what I could only assume was spacing, so after some trial and error where I'd add a space or a tab (which I entertained for too long), just decided to normalize everything